routers/web/repo/actions/actions.go

```diff
@@ -23,6 +23,12 @@ const (
 	tplViewActions base.TplName = "repo/actions/view"
 )
 
+type WorkFlow struct {
```

Comment:
Why not `Workflow`?

Just a question, not blocking.

Reply:
You are right. `WorkFlow` is strange.